packages/zowe-explorer-api/src/fs/utils/jobs.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { IJobFile } from "@zowe/zos-jobs-for-zowe-sdk";
+import { IFileSystemEntry, JobEntry, SpoolEntry } from "../types";
+import { FileType } from "vscode";
+
+export function isJobEntry(entry: IFileSystemEntry): entry is JobEntry {
+    return entry != null && entry.type == FileType.Directory;
```

Comment:
Similar question to above - wouldn't this return true for a USS dir entry too?

Reply:
Yep - thankfully `instanceof` saves the day here as well 😂  resolved in upcoming commit